### ISSUE TITLE
Mobiflight does not allow loading an empty config to a board

### ISSUE THREAD

**Issue by JaimeLeon2** [MobiFlight/MobiFlight-Connector#1224]
**Describe the bug**
After creating and deleting one single device, Mobiflight will not allow uploading an empty config to a board.

**Steps To Reproduce**
1. Upload firmware to a new board
2. Create one button in device list
3. Upload config
4. Now delete button
5. Note MF does not allow to upload this empty config to the board.

**Expected behavior**
As the current config is different from the EEPROM config, Mobiflight should allow uploading the empty config.  The Upload config button should be active.

**Actual behavior**
Upload button remains deactivated.  
Only course of action to have an empty board is to factory reset and reload the firmware, which seems too extreme.

**Details**
 - MobiFlight Version: 9.7.1.1
 - FlightSim: irrelevant
 - AddOn: irrelevant

**Additional context**
None

**Comment by DocMoebiuz** [comment]
True. I do the same workaround. There is no real reason one could not upload an empty config.

**Comment by neilenns** [comment]
I took a quick pass at trying to fix this and didn't get very far. I thought it would be as simple as removing the `moduleNode.Nodes.Count > 0` part of this:

https://github.com/MobiFlight/MobiFlight-Connector/blob/2574dc07a67a519d0a8bc4caed601c89cdfa94fc/UI/Panels/Settings/MobiFlightPanel.cs#L242

But it wasn't reliable. Sometimes it would still be enabled, other times it wouldn't. Couldn't figure out why. Gave up.
